pom.xml

```diff
@@ -149,12 +149,27 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>de.retest</groupId>
+			<artifactId>recheck-junit-jupiter-extension</artifactId>
+			<version>1.6.0-beta.1</version>
```

Comment:
Will have to update both versions _after_ their stable release but _before_ merging this PR.